### PR TITLE
🙈 Update .gitignore file

### PR DESCRIPTION
**Pull request by rampoe**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,16 +1,94 @@
...
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+env/
```
Comment (Hojagulyyev) [inline .gitignore:35]: in the readme.md file the name of project's virtual environment is venv
```
virtualenv venv
```

File: .gitignore [hunk 2]
```diff
@@ -1,16 +1,94 @@
...
+*.log
 *.pyc
-__pycache__
+local_settings.py
 db.sqlite3
-logs
 
-# enviroments
+# Environments
+.venv
```
Comment (Hojagulyyev) [inline .gitignore:86]: it was a virtual environment folder so you can delete it

**Review by Hojagulyyev — CHANGES_REQUESTED** [review]

**Review by Hojagulyyev — APPROVED** [review]

**Comment by Hojagulyyev** [comment]
@rampoe I missed the name of your commits,
commit's name: 🙈 Update .gitignore file

why don't you use conventional commits?
https://www.conventionalcommits.org/en/v1.0.0/

**Comment by Hojagulyyev** [comment]
> @rampoe I missed the name of your commits, commit's name: 🙈 Update .gitignore file
> 
> why don't you use conventional commits? https://www.conventionalcommits.org/en/v1.0.0/

how about 
refactor: update .gitignore file

**Comment by rampoe** [comment]
> Why don't you use conventional commits?  https://www.conventionalcommits.org/en/v1.0.0/

Honestly, I didn't know about them.

> how about
> refactor: update .gitignore file

Can I go with
refactor: 🙈 Update .gitignore file
like here: https://github.com/folke/devmoji

**Comment by Hojagulyyev** [comment]
do you like using emojis in commits?
if yes then you can use your style

**Comment by rampoe** [comment]
Alright let's go with your one I don't think having emojis in some commits and avoiding them in others is a good idea.

**Pull request merged**